### PR TITLE
update README and enviroment install script

### PR DESCRIPTION
**Pull request by meiyanz**
update README and enviroment install script

### CODE REVIEW

File: doc/Start_CDN_Transcode_Sample.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Please choose the transcode mode of the live0 ([hw]: hardware is for E3/VCA2 or [sw]: software is for E5): sw
+Do you need to deploy live-transcode-service by customizing parameters([y] or [n]): y
+Please choose the output channel (1, 2 ,3, 4): 1
+Please choose the 1th output encoder ('AVC', 'HEVC', 'AV1'): avc
+Please choose the 1th output resolution (a: hd480, b: hd720, c: hd1080, d: 2kqhd): d
+Please enter the 1th output bitrate([1-20]Mbps): 5
+Please choose the 1th output streaming media communication protocol(a: HLS, b: DASH): a
+Please enter the 1th output video clip name: media
+Do you still need to deploy the 2th live0? ([y] or [n]): n
+Please input run cdn request cpu quota: 1
```
Comment (TangZhiZhen) [inline doc/Start_CDN_Transcode_Sample.md]: this hint is out of date, please update it.

File: README.md [hunk 2]
```diff
@@ -32,5 +33,70 @@ The CDN Transcode Sample may run on Linux* 64 bit operating systems. The list be
...
+	make
+```
+
+
+
+
+## Deploy
+### Auto deployment using Kubernetes
+
+It divide into two parts: master node or slave node
```
Comment (wenquan-mao) [inline README.md]: It divides...

File: script/install_dependency.sh [hunk 3]
```diff
@@ -71,7 +71,9 @@ elif [ "$LINUX_DISTRO" == "CentOS" ]; then
     try_command chmod +x /usr/bin/kompose
     try_command yum install -y epel-release 
     try_command yum install -y python36 python36-pip python3-devel gtk3-devel
-    try_command pip3 install ruamel.yaml fabric3 wxpython
+    #try_command pip3 install ruamel.yaml fabric3
```
Comment (TangZhiZhen) [inline script/install_dependency.sh]: please remove it

Comment (meiyanz) [reply script/install_dependency.sh]: remove it.

**Pull request merged**